src/Kunstmaan/GeneratorBundle/Generator/KunstmaanGenerator.php

```diff
@@ -269,7 +279,12 @@ protected function installDefaultPageTemplates($bundle)
     protected function installDefaultPagePartConfiguration($bundle)
     {
         // Pagepart configuration
-        $dirPath = sprintf('%s/Resources/config/pageparts/', $bundle->getPath());
+        if (Kernel::VERSION_ID >= 40000) {
+            $dirPath = sprintf('%s/../config/kunstmaancms/pageparts/', $bundle->getPath());
```

Comment:
```suggestion
                $dirPath = $this->container->getParameter('kernel.project_dir') . '/config/kunstmaancms/pageparts/';
```

To keep the code inline with the check in [`KunstmaanGenerateCommand`](https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/7b48b9c1d0bcecca7970011e8ef46d818d3a9ebb/src/Kunstmaan/GeneratorBundle/Command/KunstmaanGenerateCommand.php#L351)